Securitytxt/Model/Config/Signature.php

```diff
@@ -24,7 +38,7 @@ class Signature implements CommentInterface
      */
     public function getCommentText($elementValue): string
     {
-        return "<a href='https://devdocs.magento.com/' target='_blank'>
+        return "<a href='{$this->instructionLink}' target='_blank'>
```

Comment:
Please use `escapeUrl` and change to not to show anything if `instructionLink` is empty.
Make sure to use translation as well.